cypress/e2e/BiographyTestsFrontEnd/index.js

```diff
@@ -0,0 +1,62 @@
+/// <reference types="Cypress" />
+import { Given, And, Then } from 'cypress-cucumber-preprocessor/steps';
+
+Then('page title is {string}',(title)=>{
+cy.get(`h1:contains("${title}")`).should('be.visible').and('include.text',title)
+})
+
+And('the individuals title is {string}',(chiefCommunicationTeam)=>{
+cy.get(`div:contains("${chiefCommunicationTeam}")`).should('be.visible').and('include.text',chiefCommunicationTeam)
+})
+
+And('the following organizations are listed for this individual',()=>{
+cy.get('.profile-org').should('be.visible')
+cy.get('.profile-org').should('be.visible')
+})
+
+And('individuals office is at {string}',(nciCompus)=>{
+cy.get("div[class='profile-panel-content'] p").contains(nciCompus).should('be.visible');
+})
+
+And('contact phone number is {string}',(phoneNumber)=>{
+cy.get('.profile-contact-info').contains(phoneNumber).should('be.visible');
+})
+
+And('contact email is {string}',(contactEmail)=>{
+cy.get(`a:contains("${contactEmail}")`).should('be.visible')
+})
+
+And('the following social media links are present',()=>{  
+cy.get('.profile-panel-social-media').should('be.visible')
+})
+
+And('biography description contains text {string}',(followLoukissasOnTwitter)=>{
+cy.get("div[id='cgvBody'] p:nth-child(1)").should('be.visible',followLoukissasOnTwitter)
```

Comment:
so this is passing for you, because cypress assertion is used incorrectly - look at what text is located by your selector, it DOES NOT contain the text you need.  the assertion should be - .should('be.visible').and("include.text", followLoukissasOnTwitter)
<img width="1249" alt="Screen Shot 2022-10-19 at 10 38 07 AM" src="https://user-images.githubusercontent.com/67657582/196722323-dc7ea8c2-52ad-4792-a000-c93c5de86ff5.png">

Reply:
also, locator is obviously wrong, it grabs wrong paragraph